.github/workflows/build.yml

```diff
@@ -16,6 +16,7 @@ on:
       - 'package.json'
       - 'package-lock.json'
       - 'tsconfig.json'
+      - '.nvmrc'
```

Comment:
❓ for consistency, I suggest to move the entry above to keep the "folder first then files in alphabetical order" implicit rule.

Reply:
Ok, to move it ^^